### PR TITLE
CAD-2069: Tracing changes supporting K=1000, batch 1

### PR DESCRIPTION
**Pull request by deepfire**
1. update `iohk-monitoring` to `master`
1. compact & frequent reporting of process statistics
1. optional `.eventlog` collection, via specially-linked `cardano-node-eventlogged` Nix attribute & `eventlog` node NixOS service flag

### CODE REVIEW

File: nix/pkgs.nix [hunk 1]
```diff
@@ -48,6 +59,7 @@ pkgs: _: with pkgs;
   inherit (cardanoNodeHaskellPackages.cardano-cli.components.exes) cardano-cli;
   inherit (cardanoNodeHaskellPackages.bech32.components.exes) bech32;
   cardano-node-profiled = cardanoNodeProfiledHaskellPackages.cardano-node.components.exes.cardano-node;
+  cardano-node-evenlogged = cardanoNodeEventlogHaskellPackages.cardano-node.components.exes.cardano-node;
```
Comment (intricate) [inline nix/pkgs.nix:62]: Is this a typo?

```suggestion
  cardano-node-eventlogged = cardanoNodeEventlogHaskellPackages.cardano-node.components.exes.cardano-node;
```

Comment (deepfire) [reply nix/pkgs.nix:62]: Right, thank you Luke!

**Review by intricate — APPROVED** [review]

**Pull request merged**